packages/rum-core/src/domain/view/viewMetrics/trackInteractionToNextPaint.ts

```diff
@@ -77,7 +76,20 @@ export function trackInteractionToNextPaint(
         interactionToNextPaintTargetSelector = undefined
       }
     }
-  })
+  }
+
+  const firstInputSubscription = createPerformanceObservable(configuration, {
+    type: RumPerformanceEntryType.FIRST_INPUT,
+    buffered: true,
+  }).subscribe(handleEntries)
+
+  const eventSubscription = createPerformanceObservable(configuration, {
+    type: RumPerformanceEntryType.EVENT,
+    // durationThreshold only impact PerformanceEventTiming entries used for INP computation which requires a threshold at 40 (default is 104ms)
+    // cf: https://github.com/GoogleChrome/web-vitals/blob/3806160ffbc93c3c4abf210a167b81228172b31c/src/onINP.ts#L209
```

Comment:
🥜 Nit: This is not a permanent link. Maybe we cf to this: https://github.com/GoogleChrome/web-vitals/blob/3806160ffbc93c3c4abf210a167b81228172b31c/src/onINP.ts#L202-L210